trace_decoder/src/typed_mpt.rs

```diff
@@ -272,6 +275,9 @@ impl StateTrie {
     pub fn as_hashed_partial_trie(&self) -> &mpt_trie::partial_trie::HashedPartialTrie {
         self.typed.as_hashed_partial_trie()
     }
+    pub fn as_mut_hashed_partial_trie_unchecked(&mut self) -> &mut HashedPartialTrie {
```

Comment:
Very nit: why `unchecked`?

Reply:
The user could insert garbage into the True, so a get would fail, because it expects well-encoded RLP